src/builder/module.rs

```diff
@@ -55,6 +55,7 @@ impl From<elements::Module> for ModuleScaffold {
 		let mut code: Option<elements::CodeSection> = None;
 		let mut data: Option<elements::DataSection> = None;
 
+        let mut other = Vec::new();
```

Comment:
Some funny whitespace business going on here.

Reply:
It seems the source is using tab, and here is spaces